src/test/java/seedu/duke/ViewFunctionTest.java

```diff
@@ -68,84 +73,699 @@ void view_allFieldsValid_success() {
+                    + System.lineSeparator()
+                    + "Expense | ENTERTAINMENT | 2019-06-06 |   Maid Cafe    |-$14.60      | YEAR  | 2021-11-06"
+                    + System.lineSeparator()
+                    + "Expense | ENTERTAINMENT | 2018-06-06 |   Maid Cafe    |-$14.60      | YEAR  | 2021-11-06"
+                    + System.lineSeparator()
+                    + "Expense | ENTERTAINMENT | 2017-06-06 |   Maid Cafe    |-$14.60      | YEAR  | 2021-11-06"
+                    + System.lineSeparator()
+                    + "Expense | ENTERTAINMENT | 2016-06-06 |   Maid Cafe    |-$14.60      | YEAR  | 2021-11-06"
+                    + System.lineSeparator()
+                    + "Expense | ENTERTAINMENT | 2015-06-06 |   Maid Cafe    |-$14.60      | YEAR  | 2021-11-06"
+                    + System.lineSeparator()
+                    + "Expense | ENTERTAINMENT | 2014-06-06 |   Maid Cafe    |-$14.60      | YEAR  | 2021-11-06"
+                    + System.lineSeparator()
+                    + "Expense | ENTERTAINMENT | 2013-06-06 |   Maid Cafe    |-$14.60      | YEAR  | 2021-11-06"
+                    + System.lineSeparator()
+                    + "Expense | ENTERTAINMENT | 2012-06-06 |   Maid Cafe    |-$14.60      | YEAR  | 2021-11-06"
+                    + System.lineSeparator()
+                    + "Expense |     FOOD      | 2021-02-01 | Samurai Burger |-$7.50       |       |"
+                    + System.lineSeparator()
+                    + "                                           Net Total: | $251,346.50"
+                    + System.lineSeparator()
+                    + "Here is the list of all recurring entries, where some were added to the above list:"
+                    + System.lineSeparator()
+                    + "Income  |  COMMISSION   | 2021-06-09 |    OnlyFans    | $300.00     | MONTH | 2021-11-06"
+                    + System.lineSeparator()
+                    + "Expense | ENTERTAINMENT | 2012-06-06 |   Maid Cafe    |-$14.60      | YEAR  | 2021-11-06"
+                    + System.lineSeparator();
+            assertEquals(expectedOutput, output.toString());
+        } catch (MintException e) {
+            e.printStackTrace();
+        }
+    }
+
+    @Test
+    void viewByDateAscendingWithRecurringEntries_allFieldsValid_success() {
+
+        NormalFinanceManager normalFinanceManager = new NormalFinanceManager();
+        RecurringFinanceManager recurringFinanceManager = new RecurringFinanceManager();
+        BudgetManager budgetManager = new BudgetManager();
+        NormalListDataManager normalListDataManager = new NormalListDataManager();
+        DataManagerActions dataManagerActions = new DataManagerActions();
+        RecurringListDataManager recurringListDataManager = new RecurringListDataManager();
+        BudgetDataManager budgetDataManager = new BudgetDataManager();
+        Ui ui = new Ui();
```

Comment:
Perhaps you can place them outside the method since you will reuse them again in other test methods also?